### PR TITLE
Use TangoSourceDistribution to build tango-cs docker image

### PR DESCRIPTION
**Pull request by Ingvord**
This PR is intend to change the current build process to use latest TangoSourceDistribution. The following must be done to achieve this:

- [x] add mysql as service to travis
- [x] fetch TangoSourceDistribution
- [x] create intermediate build docker
- [x] build inside intermediate docker
- [x] create target docker
- [x] deploy to docker hub using tag from TangoSourceDistribution

### CODE REVIEW

**Comment by Ingvord** [comment]
In fact this repo can be merged into [tango-controls/TangoDatabase](https://github.com/tango-controls/TangoDatabase)

**Comment by Ingvord** [comment]
In fact the true docker approach would be to split this one into separate dockers aka TangoMysql/TangoDB/Starter/TAC/TangoTest. Not sure Starter will make much sense in this case. Maybe we can keep DB, Starter and TAC in a single docker. Then we basically need to merge this one and #4. And produce TangoTest docker independantly.

File: .travis.yml [hunk 1]
```diff
@@ -1,47 +1,37 @@
...
-  - >
-    ./scala-2.11.7/bin/scala -cp JTango-9.0.7.jar
-    -e "new fr.esrf.TangoApi.DeviceProxy(\"$DEVICE\")"
+  - docker build -t tangocs/tango-cs .
+
+after_script:
+  - docker rm -f tango-build
+
+before_deploy:
+  - docker login -u ingvord -p "$REGISTRY_PASS"
```
Comment (bourtemb) [inline .travis.yml]: What about replacing 
-u ingvord
with
-u "$REGISTRY_USER" 
as done in https://github.com/tango-controls/docker-mysql/pull/3/files ?

Comment (Ingvord) [reply .travis.yml]: @bourtemb , done in fbab717

**Pull request merged**